src/output-json-ike.c

```diff
@@ -63,21 +66,16 @@ typedef struct LogIKELogThread_ {
 static int JsonIKELogger(ThreadVars *tv, void *thread_data, const Packet *p, Flow *f, void *state,
         void *tx, uint64_t tx_id)
 {
-    IKETransaction *iketx = tx;
     LogIKELogThread *thread = thread_data;
-
     JsonBuilder *jb = CreateEveHeader((Packet *)p, LOG_DIR_PACKET, "ike", NULL);
     if (unlikely(jb == NULL)) {
         return TM_ECODE_FAILED;
     }
 
-    EveAddCommonOptions(&thread->ikelog_ctx->cfg, p, f, jb);
```

Comment:
Why do we remove this line ?

Reply:
See above, `cfg` was removed. I am not sure why it was removed, though. If there is nothing in the way of reinstating it, I would be happy to do so.